sormas-api/src/main/java/de/symeda/sormas/api/user/UserRight.java

```diff
@@ -1047,6 +1047,32 @@
 	),
 	CAMPAIGN_FORM_DATA_DELETE(
 			ADMIN
+	),
+	CAMPAIGN_DIAGRAM_VIEW(
```

Comment:
Let's remove those user rights until we have a user interface that allows to view, edit and delete the diagrams. For now all of this will be done directly in the database.